monai/transforms/utility/array.py

```diff
@@ -615,3 +616,32 @@ def __call__(
         )
 
         return np.concatenate([img, points_image], axis=0)
+
+
+class TorchVision:
+    """
+    This is a wrapper transform for PyTorch TorchVision transform based on the specified transform name and args.
+    As most of the TorchVision transforms only work for PIL image and PyTorch Tensor, this transform expects input
+    data to be PyTorch Tensor, users can easily call `ToTensor` transform to convert a Numpy array to Tensor.
+
+    """
+
+    def __init__(self, name: str, *args, **kwargs) -> None:
+        """
+        Args:
+            name: The transform name in TorchVision package.
+            args: parameters for the TorchVision transform.
+            kwargs: parameters for the TorchVision transform.
+
+        """
+        super().__init__()
+        transform, _ = optional_import("torchvision.transforms", name=name)
+        self.trans = transform(*args, **kwargs)
+
+    def __call__(self, img: torch.Tensor) -> torch.Tensor:
```

Comment:
it seems torch vision doesn't have proper type hints for tensor/pil object, perhaps we could remove the typing for now?

Reply:
Sure, updated in the latest commit.
Could you please help review it again?

Thanks.